docs/quickstart-config.yaml

```diff
@@ -1,19 +1,14 @@
+httpConfig:
```

Comment:
I think we can rename and then reuse it for all key-value config like config.properties and node.properties in trino.  But we can do that in a follow up.